src/components/htmlMediaHelper.js

```diff
@@ -131,8 +131,8 @@ import { Events } from 'jellyfin-apiclient';
     }
 
     function setCurrentTimeIfNeeded(element, seconds) {
-        if (Math.abs(element.currentTime || 0, seconds) <= 1) {
-            element.currentTime = seconds;
+        if (Math.abs(element.currentTime || 0) - seconds <= 1) {
+            setTimeout(function(element, seconds){ element.currentTime = seconds; }, 500, element, seconds);
```

Comment:
I can't test this right now, but I believe that you just need to schedule seeking right after current event.
```suggestion
            setTimeout(() => {
                element.currentTime = seconds;
            }, 0);
```
Maybe also leave a comment.